scene/3d/spring_arm_3d.cpp

```diff
@@ -133,17 +134,32 @@ void SpringArm3D::process_spring() {
 	Vector3 motion;
 	const Vector3 cast_direction(get_global_transform().basis.xform(Vector3(0, 0, 1)));
 
+	motion = Vector3(cast_direction * (spring_length));
+
 	if (shape.is_null()) {
-		motion = Vector3(cast_direction * (spring_length));
-		PhysicsDirectSpaceState3D::RayResult r;
-		bool intersected = get_world_3d()->get_direct_space_state()->intersect_ray(get_global_transform().origin, get_global_transform().origin + motion, r, excluded_objects, mask);
-		if (intersected) {
-			real_t dist = get_global_transform().origin.distance_to(r.position);
-			dist -= margin;
-			motion_delta = dist / (spring_length);
+		Camera3D *camera = nullptr;
+		for (int i = get_child_count() - 1; 0 <= i; --i) {
```

Comment:
What is the reason for interating backwards over cameras? If it's necessary, I think `i >= 0` is better than `0 <= i`.

Reply:
No idea, the code was like this before, so I left it as-is.

Reply:
Was it? I can't find `0 <= i` in any of the removals in this PR.

Reply:
It's copied from just below:
```
scene/3d/spring_arm_3d.cpp
154:    for (int i = get_child_count() - 1; 0 <= i; --i) {
```